tools/README.md

```diff
@@ -0,0 +1,25 @@
+Tools
+=====
+
+KA Customizer
+-------------
+KA Custimizer is a small GUI tool to tweak modules and themes of a knowledge
+app.
+
+#### Installing
+To install, run open a terminal and copy and enter the following command.
+```
+bash -exc "$(wget https://raw.githubusercontent.com/endlessm/eos-knowledge-lib/master/tools/ka-customizer-installer -O -)"
+```
+Installing may take a while! The bleeding edge versions of our flatpak runtimes
```

Comment:
That was my attempt to be less jargon-y then master branch. "Nightly unstable" perhaps?